src/lib/sw_engine/tvgSwCommon.h

```diff
@@ -313,6 +313,7 @@ void rleFree(SwRleData* rle);
 void rleReset(SwRleData* rle);
 void rleClipPath(SwRleData *rle, const SwRleData *clip);
 void rleClipRect(SwRleData *rle, const SwBBox* clip);
+void shapeMaskingAlpha (SwShape *compShape, SwRleData *rle, const SwRleData *clip);
```

Comment:
guess it has to be rleAlphaMask() for consistency?

Reply:
updated, thank you